Example/CNI-Itineraries Example.xcodeproj/xcshareddata/xcschemes/CNI-Booking-Example.xcscheme

```diff
@@ -40,27 +40,46 @@
       buildConfiguration = "Debug"
       selectedDebuggerIdentifier = "Xcode.DebuggerFoundation.Debugger.LLDB"
       selectedLauncherIdentifier = "Xcode.DebuggerFoundation.Launcher.LLDB"
-      language = ""
       shouldUseLaunchSchemeArgsEnv = "YES">
       <Testables>
          <TestableReference
             skipped = "NO">
             <BuildableReference
                BuildableIdentifier = "primary"
                BlueprintIdentifier = "607FACE41AFB9204008FA782"
-               BuildableName = "CNI-Booking_Tests.xctest"
-               BlueprintName = "CNI-Booking_Tests"
-               ReferencedContainer = "container:CNI-Booking.xcodeproj">
+               BuildableName = "CNI-Itineraries Example_Tests.xctest"
+               BlueprintName = "CNI-Itineraries Example_Tests"
+               ReferencedContainer = "container:CNI-Itineraries Example.xcodeproj">
+            </BuildableReference>
+         </TestableReference>
+         <TestableReference
+            skipped = "NO">
+            <BuildableReference
+               BuildableIdentifier = "primary"
+               BlueprintIdentifier = "3B34B22121A40FF30097FCB7"
+               BuildableName = "CNI-BookingTests.xctest"
```

Comment:
Seems there still is a reference to CNI-Booking

Reply:
Nice catch! Upated in 9c54b43.